### PR TITLE
Tests scatterplot function

### LINKED ISSUE #20: Test script for scatter plot

### PR DESCRIPTION
**Pull request by iamMoid**

### CODE REVIEW

**Comment by iamMoid** [comment]
Do these tests look good enough? It is basically testing whether the sample data and its features are correctly being assigned to different attributes of the plot.

**Comment by iamMoid** [comment]
Thoughts on what more can I add?

File: tests/test_scatterplot.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+
+def test_scatterplot():
+    # check if the dataframe is a pandas dataframe
+    if not isinstance(sample_data, pd.core.frame.DataFrame):
+        raise TypeError(
+            "sample_data should be of a pandas dataframe of type: 'pandas.core.frame.DataFrame'"
+        )
+
+def test_enc_x_shorthand:
```
Comment (ruben1dlg) [inline tests/test_scatterplot.py]: I think you are missing the brakets at the end of the name of the function. 
`def test_enc_x_shorthand():`

**Review by ruben1dlg — APPROVED** [review]

File: tests/test_scatterplot.py (hunk 1 above)
Comment (iamMoid) [reply tests/test_scatterplot.py]: Yes thanks for pointing out. I realized when I ran pytest. Updated the script now.

**Review by shyan0903 — APPROVED** [review]

**Pull request merged**